credits.cfg

```diff
@@ -131,7 +145,10 @@ Elvish Pillager
 Raajal
 ZhayTee - wesnoth_tragedy
 amilcare - intro
-PANotUncleDave - Flight_of_the_Drakes" #wmllint: no spellcheck
+PANotUncleDave - Flight_of_the_Drakes
```

Comment:
If the campaign is being changed to use the UMC Book 10 copy of this instead, does it still need to be credited here as well?

Reply:
We do not - removed this line.